### DISCUSSION TITLE
Copilot X Early Access for Build 2023 Attendees

### DISCUSSION THREAD

**Discussion by TonyLandi-Insight** [community/community#61756] [Copilot Conversations] [answered]
### Select Topic Area

Question

### Body

Hi, I was in-person at Build this year and it was announced that attendees would get early access to Copilot X. How do I claim this?

**Comment by Akash1134** [comment, marked as answer]
Hi @TonyLandi-Insight ,

Registration for Microsoft Build [was required](https://github.com/orgs/community/discussions/53945#:~:text=Last%20month%2C%20we,build.microsoft.com/) for you to get early access to GitHub Copilot X features. 

If you did registered, please check your email inbox from the GitHub team with title (GitHub Copilot X)

Thanks!
